cmd/main.go

```diff
@@ -35,6 +38,33 @@ func main() {
 		err = errors.Wrapf(err, msg)
 		log.Fatal(err)
 	}
+	res1 := string(result)
+
+	var result1 periodic.RunnerResults
+	var bd []byte
+
+	hres, _ := res1.(*fhttp.HTTPRunnerResults)
```

Comment:
Hopefully, this type assertion doesn't rear its ugly head.

Reply:
That is what it does in the upstream repo which ultimately caused errors in the tests and whenever there was an error in nighthawk, due to the load test interface the test fell back to fortio which is why the tests were successful each and every time.